CITATION.cff

```diff
@@ -0,0 +1,113 @@
+# This file is a citation metadata file in the Citation File Format
+# (http://citation-file-format.github.io/).
+#
+# It provides citation metadata for this version of stella
+
+cff-version: 1.1.0
+message: If you use stella in your work, please cite it using the following metadata.
+title: stella
+authors:
+  - family-names: Barnes
+    given-names: Michael
+    affiliation: University of Oxford
+    orcid: https://orcid.org/0000-0002-0177-1689
+
+  - family-names: St-Onge
+    given-names: Denis
+    affiliation: University of Oxford
+    orcid: https://orcid.org/0000-0003-3112-9221
+
+  - family-names: Dickinson
+    given-names: David
+    orcid: https://orcid.org/0000-0002-0868-211X
+    affiliation: University of York
+
+  - family-names: Dorland
+    given-names: William
+    orcid: https://orcid.org/0000-0003-2915-724X
+    affiliation: University of Maryland
+
+  - family-names: Hill
+    given-names: Peter Alec
+    orcid: https://orcid.org/0000-0003-3092-1858
+    affiliation: University of York
+
+  - family-names: Parker
+    given-names: Joseph Thomas
+    orcid: https://orcid.org/0000-0002-5573-6475
+    affiliation: United Kingdom Atomic Energy Authority
+
+  - family-names: Roach
+    given-names: Colin Malcolm
+    affiliation: Culham Centre for Fusion Energy
+
+  - family-names: Landreman
+    given-names: Matt
+    orcid: https://orcid.org/0000-0002-7233-577X
+
+  - family-names: Thienpondt
+    given-names: Hanne
+
+  - family-names: Numata
+    given-names: Ryusuke
+
+  - family-names: Parisi
+    given-names: Jason
+    affiliation: University of Oxford
+    orcid: https://orcid.org/0000-0003-1328-7154
+
+  - family-names: Wilkie
+    given-names: George
+    orcid: https://orcid.org/0000-0002-6826-6273
+
+  - family-names: Anton
+    given-names: Lucian
+
+  - family-names: Ball
+    given-names: Justin
+
+  - family-names: Baumgaertel
+    given-names: Jessica
+
+  - family-names: Colyer
+    given-names: Greg
+
+  - family-names: Hardman
+    given-names: Michael
+
```

Comment:
suggest removing

Reply:
He does have one commit in stella: c9e3eb4

Reply:
This seems like just some changes to the makefile to fix compilation, and I may have told him to make these changes.